GLideN64/src/Graphics/OpenGLContext/GLFunctions.cpp

```diff
@@ -398,4 +410,8 @@ extern "C" void initGLFunctions()
 
 	GL_GET_PROC_ADR(PFNGLEGLIMAGETARGETTEXTURE2DOESPROC, EGLImageTargetTexture2DOES);
 	GL_GET_PROC_ADR(PFNGLEGLIMAGETARGETRENDERBUFFERSTORAGEOESPROC, EGLImageTargetRenderbufferStorageOES);
+
+#if defined(USE_DLSYM_FOR_GL)
+	dlclose(dlhandle);
```

Comment:
Does this actually work?
Probably only because frontend still has a handle, theoretically this would unload the GLES lib if all other handles are closed.

Reply:
Yes yes, I tested it of course.
Nothing theoretic, that is exactly how it works. `dlclose()` just decreases usage count number.